services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/IntentionalRerouteSpec.groovy

```diff
@@ -187,6 +187,64 @@ class IntentionalRerouteSpec extends BaseSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    def "Should be able to reroute to a path with not enough bandwidth available in case ignoreBandwidth=true"() {
+        given: "A flow with alternate paths available"
+        def switches = topology.getActiveSwitches()
+        List<List<PathNode>> allPaths = []
+        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
+                .findAll { src, dst -> src != dst }.unique { it.sort() }.find { Switch src, Switch dst ->
+            allPaths = database.getPaths(src.dpId, dst.dpId)*.path
+            allPaths.size() > 1
+        } ?: assumeTrue("No suiting switches found", false)
+        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flow.maximumBandwidth = 10000
+        flow.ignoreBandwidth = true
+        flowHelper.addFlow(flow)
+        def currentPath = PathHelper.convert(northbound.getFlowPath(flow.id))
+
+        when: "Make the current path less preferable than alternatives"
+        def alternativePaths = allPaths.findAll { it != currentPath }
+        alternativePaths.each { pathHelper.makePathMorePreferable(it, currentPath) }
+
+        and: "Make all alternative paths to have not enough bandwidth to handle the flow"
+        def currentIsls = pathHelper.getInvolvedIsls(currentPath)
+        def newBw = flow.maximumBandwidth - 1
+        def changedIsls = alternativePaths.collect { altPath ->
+            def thinIsl = pathHelper.getInvolvedIsls(altPath).find {
+                !currentIsls.contains(it) && !currentIsls.contains(it.reversed)
+            }
+            [thinIsl, thinIsl.reversed].each {
+                database.updateIslMaxBandwidth(it, newBw)
+                database.updateIslAvailableBandwidth(it, newBw)
+            }
+            thinIsl
+        }
+
+        and: "Init a reroute to a more preferable path"
+        def rerouteResponse = northbound.rerouteFlow(flow.id)
+
+        then: "The flow is rerouted because ignoreBandwidth=rue"
+        int seqId = 0
+
+        rerouteResponse.rerouted
+        rerouteResponse.path.path != currentPath
+        rerouteResponse.path.path.each { assert it.seqId == seqId++ }
+
+        and: "Available bandwidth was not changed while rerouting due to ignoreBandwidth=true"
+        def updatedPath = PathHelper.convert(northbound.getFlowPath(flow.id))
+        updatedPath != currentPath
+        Wrappers.wait(WAIT_OFFSET) { assert northbound.getFlowStatus(flow.id).status == FlowState.UP }
```

Comment:
L234-L236 belong to `then: "The flow is rerouted"`, these verifications are not related to bandwidth at all

Reply:
I've just tried to store the `updatedPath` variable more close.
Fixed